src/ResourceManager/Common/Commands.ScenarioTests.ResourceManager.Common/EnvironmentSetupHelper.cs

```diff
@@ -53,8 +54,14 @@ public class EnvironmentSetupHelper
         private AzureAccount testAccount;
 
         private const string PackageDirectoryFromCommon = @"..\..\..\..\Package\Debug";
+#if !NETSTANDARD
+
         public string PackageDirectory = @"..\..\..\..\..\Package\Debug";
         public string StackDirectory = @"..\..\..\..\..\..\Stack\Debug";
+#else
```

Comment:
Why is netcore using release by default and netframework using debug? Might be easier to make it generic (non-netcore/framework specific).